sb_vision/cvcapture.py

```diff
@@ -97,31 +97,3 @@ def close(self) -> None:
                     self.instance = None
 
     __del__ = close
-
-
-def clean_and_threshold(image: bytes, width: int, height: int) -> bytes:
-    """Prepare an image (as bytes) with thresholding/filtering."""
-    if len(image) != width * height:
-        raise ValueError("image has the wrong length")
-
-    source_buffer = _cvcapture.ffi.new(
-        'uint8_t[{}]'.format(width * height),
-    )
-    dst_buffer = _cvcapture.ffi.new(
-        'uint8_t[{}]'.format(width * height),
-    )
-
-    _cvcapture.ffi.memmove(
-        source_buffer,
-        image,
-        len(image),
-    )
-
-    _cvcapture.lib.cvthreshold(
```

Comment:
Does this mean we can also remove the native function which provides this?

Reply:
Reading through, it looks like we can, yes 🎉

Reply:
Sorry, I wasn't quite sure what this meant. Think I've done it now.